csrc/options.cpp

```diff
@@ -261,6 +261,15 @@ const std::vector<std::string>& getDebugDumpArguments(DebugDumpOption option) {
   return DebugDumpOptionsGuard::getCurOptions().getArgs(option);
 }
 
+bool hasDebugDumpArgument(DebugDumpOption option, const std::string& arg) {
```

Comment:
This could be a new method `bool Options::hasArgument(OptionEnum option, const std::string& arg) const` and we can call it here as
```
bool hasDebugDumpArgument(DebugDumpOption option, const std::string& arg) {
  return DebugDumpOptionsGuard::getCurOptions().hasArgument(option, arg);
}
```